src/domain/feeder_actuator.py

```diff
@@ -13,28 +13,31 @@ class FeederActuator(Actuator):
     class Commands(Enum):
         OPEN = json.dumps({"open": True})
 
+    def __init__(self, id, enabled):
+        super().__init__(id, enabled)
+        self.type = Actuator.ActuatorTypes.FEEDER
+
     def send_command(self, cmd):
         """Method to send the specific command to the Actuator superclass."""
         if isinstance(cmd, FeederActuator.Commands):
             super().send_command(cmd)
         else:
             logger.error(
                 "Actuator %s with ID %s received an unknown command: %s.",
-                self.__class__.__name__,
-                self.actuator_id,
+                self.type,
+                self.id,
                 cmd,
             )
             raise Exception("Unknown command")
 
     def serialize(self):
         """Method to serialize Actuator object into file."""
         return {
-            "type": self.__class__.__name__,
-            "id": self.actuator_id,
+            "id": self.id,
             "enabled": self.enabled,
         }
 
     @staticmethod
     def deserialize(data):
         """Method to deserialize Actuator object from file."""
-        return FeederActuator(**data)
+        return FeederActuator(data["id"], data["enabled"])
```

Comment:
**data didn't work?

Reply:
We don't have to serialize the type as the type is defined in init() and not modifiable.